plugins/org.eclipse.elk.alg.rectpacking/src/org/eclipse/elk/alg/rectpacking/RectPackingLayoutProvider.java

```diff
@@ -70,6 +71,8 @@ public void layout(final ElkNode layoutGraph, final IElkProgressMonitor progress
         boolean expandToAspectRatio = layoutGraph.getProperty(RectPackingOptions.EXPAND_TO_ASPECT_RATIO);
         // Whether interactive layout is activ.
         boolean interactive = layoutGraph.getProperty(RectPackingOptions.INTERACTIVE);
+        // A target width for the algorithm. If this is set the width approximation step is skipped.
+        double targetWidth = layoutGraph.getProperty(RectPackingOptions.TARGET_WIDTH);
```

Comment:
It's just a minor detail, but personally, I would prefer not relying on default values that actually mean "this property is not set", since we have a mechanism to check whether or not it is: `hasProperty(...)`. I'd skip the property's default value and initialize `targetWidth` here depending on whether the property is set or not.